package.json

```diff
@@ -49,7 +51,7 @@
 		"@woocommerce/e2e-environment": "^0.2.3",
 		"@woocommerce/e2e-utils": "^0.1.6",
 		"@woocommerce/eslint-plugin": "^1.2.0",
-		"@wordpress/dependency-extraction-webpack-plugin": "^2.7.0",
```

Comment:
What is the advantage again to have it remote instead of using standard  `^2.7.0` versioning?

Reply:
@puntope , it is related to the following, which is mentioned in the PR description.

> :warning:  So far this PR uses `trunk` version of `@wordpress/dependency-extraction-webpack-plugin` to be able to use unreleased https://github.com/WordPress/gutenberg/pull/35106, hopefully, we'd be able to use released version soon.